fatiando/gridder.py

```diff
@@ -439,3 +444,244 @@ def cut(x, y, scalars, area):
               if x[i] >= xmin and x[i] <= xmax
               and y[i] >= ymin and y[i] <= ymax]
     return [x[inside], y[inside], [s[inside] for s in scalars]]
+
+def pad_array(a, xy=None, np=None, padtype='OddReflectionTaper'):
```

Comment:
@drandykass it's better to avoid `np` for variable names since that's usually what people import numpy as. We've been changing this import in Fatiando lately to make code a bit nicer to read. So if you want to change the `import numpy`  to `import numpy as np` in this module, please feel free. You'd need to replace all calls to `numpy` with `np` though.